pkg/services/mssql/database_fe_update.go

```diff
@@ -0,0 +1,63 @@
+package mssql
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/open-service-broker-azure/pkg/service"
+)
+
+func (d *databaseFeManager) ValidateUpdatingParameters(
+	instance service.Instance,
+) error {
+	td := instance.Plan.GetProperties().Extended["tierDetails"]
+	details := td.(planDetails)
+	return details.validateUpdateParameters(instance)
+}
+
+func (d *databaseFeManager) GetUpdater(service.Plan) (service.Updater, error) {
+	// There isn't a need to do any "pre-provision here. just the update step"
+	return service.NewUpdater(
+		service.NewUpdatingStep("updateARMTemplate", d.updateARMTemplate),
+	)
+}
+
+func (d *databaseFeManager) updateARMTemplate(
```

Comment:
What's the behavior of updating a database? Seems `mysql-database` cannot be updated.

Reply:
It is mssql, just like *mssql/database_update.go*.